### PR TITLE
Add `og` metadata

### PR DESCRIPTION
**Pull request by thebrandonlucas**
I didn't know what twitter data to add, so please feel free to add anything I'm missing, but this adds the [4 required fields](https://www.freecodecamp.org/news/what-is-open-graph-and-how-can-i-use-it-for-my-website/#:~:text=og%3Atitle%3A%20The,the%20current%20page.) for `og` plus `og:description`.

Solves #25

### CODE REVIEW

**Review by sbddesign — APPROVED** [review]
This was a great start. I pushed some commits to add the stuff for Twitter formatting. 

Furthermore, I needed to make sure we could customize the metadata for every page (for example, every page has a different title). I found that there is a "Nextier" way tol do this [via their metadata API](https://nextjs.org/docs/app/api-reference/functions/generate-image-metadata).

So I re-jigged it to use `Metadata` and not cram it all into the `<head> ourselves. What do you think, does this look right to you?

File: app/layout.tsx [hunk 1]
```diff
@@ -1,9 +1,40 @@
...
+}
+
 export const metadata: Metadata = {
-  title: "Create Next App",
-  description: "Generated by create next app",
+  title: defaultMeta.title,
+  description: defaultMeta.description,
+  openGraph:{
+    // title: defaultMeta.title,
+    // description: defaultMeta.description,
```
Comment (thebrandonlucas) [inline app/layout.tsx:20]: Why are we commenting these out? Will they not be used?

File: app/layout.tsx [hunk 2]
```diff
@@ -1,9 +1,40 @@
...
+        alt: defaultMeta.image.alt
+      }
+    ],
+    locale: "en_US",
+    type: "website"
+  },
+  twitter: {
+    card: 'summary_large_image',
+    // title: defaultMeta.title,
+    // description: defaultMeta.description,
```
Comment (thebrandonlucas) [inline app/layout.tsx:35]: Same Q as above

**Review by thebrandonlucas — COMMENTED** [review]
Why are we commenting these out? Will they not be used?

**Pull request merged**